pkg/certs/util_test.go

```diff
@@ -0,0 +1,101 @@
+package certs
+
+import (
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"net"
+	"testing"
+	"time"
+)
+
+func init() {
+	nowFn = func() time.Time {
+		return time.Date(2019, time.January, 1, 0, 0, 0, 0, &time.Location{})
+	}
+}
+
+func TestCertificateToString(t *testing.T) {
+	tests := []struct {
+		name     string
+		cert     *x509.Certificate
+		expected string
+	}{
+		{
+			name:     "empty cert",
+			cert:     &x509.Certificate{},
+			expected: `"" [] issuer="<self-signed>" (0001-01-01 00:00:00 +0000 UTC to 0001-01-01 00:00:00 +0000 UTC (now=2019-01-01 00:00:00 +0000 UTC))`,
```

Comment:
can we remove the timeshift and timezone?

Reply:
i'm not sure why @deads2k added this (i'm just copying his code)